src/components/radioGroup/radioCard/radioCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { RadioGroupIndicator, RadioGroupItem, type RadioGroupItemProps } from '@radix-ui/react-radio-group';
+import classNames from 'classnames';
+import { forwardRef, useId } from 'react';
+import { Avatar } from '../../avatars';
+import { Icon, IconType } from '../../icon';
+import { Tag, type ITagProps } from '../../tag';
+
+export interface IRadioCardProps extends RadioGroupItemProps {
+    /**
+     * Radio card avatar image source
+     */
+    avatar?: string;
+    /**
+     * Description
+     */
+    description: string;
+    /**
+     * Radio label
+     */
+    label: string;
+    /**
+     * Radio card tag
+     */
+    tag?: ITagProps;
+}
+
+/**
+ * `RadioCard` component
+ *
+ * This component is based on the Radix-UI radio implementation.
+ * An exhaustive list of its properties can be found in the corresponding Radix primitive
+ * [documentation](https://www.radix-ui.com/primitives/docs/components/radio-group#item).
+ *
+ * **NOTE**: The component must be used inside a `<RadioGroup />` component in order to work properly.
+ */
+export const RadioCard = forwardRef<HTMLButtonElement, IRadioCardProps>((props, ref) => {
+    const { value, id, className, tag, avatar, label, description, ...rest } = props;
+
+    const randomId = useId();
+    const processedId = id ?? randomId;
+
+    const containerClasses = classNames(
+        'group h-16 rounded-xl border border-neutral-100 bg-neutral-0 px-4 py-3 md:h-20 md:rounded-2xl md:px-6 md:py-4', // default
```

Comment:
Currently the cursor switches between default and pointer when hovering hover the "label" property or "description" property. Similar effect is for the box-shadow

Reply:
Ah, it's the label in the button